testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java

```diff
@@ -202,6 +202,10 @@ public Pattern[] get(boolean inContainer) {
                         Pattern.compile(".*Exception occurred when setting value \"150/s\" for class jdk.jfr.internal.Control.*")};
             } else {
                 return new Pattern[]{
+                        // We don't support the new JFR Deprecated events annotation yet. (https://github.com/oracle/graal/pull/8037)
+                        Pattern.compile(".*Exception occurred when setting value \"forRemoval\" for class jdk.jfr.internal.Control*"),
+                        // We don't support the OldObjectSample event yet. (https://github.com/oracle/graal/pull/8037)
+                        Pattern.compile(".*Exception occurred when setting value \"0 ns\" for class jdk.jfr.internal.Control*"),
```

Comment:
I don't understand. Why do we see this message in CI, when https://github.com/oracle/graal/pull/8037 hasn't been merged yet? It must be coming from something else?

Reply:
https://github.com/oracle/graal/pull/8037 isn't what's creating those log lines, it just adds more text to make them clearer. [This recent GraalVM commit](https://github.com/oracle/graal/commit/dd8eb7436ca2ce323a74667805dd7736c6ccc972) is throwing the exception, which is caught in the JDK code which results in those error logs. This is the correct behaviour since we don't actually handle [those JFR features](https://github.com/openjdk/jdk/pull/16931) yet.

Reply:
The reason I mentioned waiting for https://github.com/oracle/graal/pull/8037 to be merged is so that we wouldn't have to update the whitelisted lines a second time with the modified log text.

Reply:
OK. Can we adjust the comments then, please? I suggest something like:

```
/*
 * We don't support the new JFR Deprecated events annotation yet.
 * https://github.com/oracle/graal/commit/dd8eb7436ca2ce323a74667805dd7736c6ccc972 throws
 * VMErrors which are handled in JDK code, and then get logged at runtime. Allow list those
 * log lines until they are supported.
 */
Pattern.compile(".*Exception occurred when setting value \"forRemoval\" for class jdk.jfr.internal.Control*"),
Pattern.compile(".*Exception occurred when setting value \"0 ns\" for class jdk.jfr.internal.Control*"),
```

For the updated messages, depending on https://github.com/oracle/graal/pull/8037, we can create a new issue. Perhaps this PR should not close https://github.com/Karm/mandrel-integration-tests/issues/237 but use a separate issue explaining the situation this fixes.

Then, once 8037 merges, update the pattern lines. It should be quick to do. We need to reduce our CI noise so as to not mask actual issues we don't know of yet.

Reply:
It's very confusing that those lines refer to a PR that is unrelated (https://github.com/oracle/graal/pull/8037). https://github.com/oracle/graal/commit/dd8eb7436ca2ce323a74667805dd7736c6ccc972 is the actual change provoking those we see in CI, right? So that should be what we refer to.

Reply:
@roberttoyonaga I've created #239 to refer to for this PR. Thanks!

Reply:
Thanks @jerboaa, I've updated the comments.